app/lib/ability.rb

```diff
@@ -308,6 +308,10 @@ def account_administrator_abilities(user, resource, controller)
       can :manage, AccountUser
       can [:show, :suspend, :unsuspend, :user_search, :user_accounts, :statements, :show_statement, :index], Statement
     end
+
+    if SettingsHelper.feature_on?(:move_transactions_account_roles) && Account.administered_by(user).any?
+      can [:movable_transactions, :reassign_chart_strings, :confirm_transactions, :move_transactions], TransactionsController
```

Comment:
is there are reason to use this granularity instead of a single `move_transactions` or something like that?

Reply:
I don't think I understood your question. You mean why using the actions or the controller?